README.md

```diff
@@ -12,9 +12,15 @@ Soapbox provides managed web application hosting services, encapsulating best-pr
  - Run `docker-compose up` to build, create and start the containers with streaming logs.
  - Run `modd` in another terminal to start the file-watcher. Any changes to local files will rebuild binaries and restart services as needed.
  - Visit [http://localhost:3000/](http://localhost:3000/) and go!
+   - Note that you'll have to set up Github OAuth (instructions below) in order to create and manage applications.
```

Comment:
Actually, you only need to set up GitHub OAuth to be able to create apps from private repos and to be able to prepopulate a list of repos and commits. So maybe just add that little caveat here.

Reply:
I wonder if I'm running into a different issue then?  When I log in as a user with no GitHub account linked, I get a Rails error when navigating to create a new application:

![image](https://user-images.githubusercontent.com/3094090/31407888-83f65006-add4-11e7-917d-028147ab96a7.png)

Is this a known issue?

Reply:
Ahhh, good catch. We should fallback gracefully and not access the GitHub API if the user hasn't OAuthed yet. I will push a fix for this, in the meantime we can merge this as is and I will change the README as part of that PR.

Reply:
Excellent, thank you sir!